generators/client/templates/vue/src/main/webapp/app/account/activate/activate.component.ts.ejs

```diff
@@ -30,6 +32,6 @@ export default class Activate extends Vue {
   }
 
   public openLogin(): void {
-      this.loginModalService().openLogin((<any>this).$root);
+      this.loginService().openLogin((<any>this).$root);
```

Comment:
the method openLogin won't exist for OAuth2

Reply:
This component should not even been here with oauth2. I changed this.